FaunaDB.Client/Client/FaunaClient.cs

```diff
@@ -32,13 +33,15 @@ public class FaunaClient
         /// <param name="endpoint">URL for the FaunaDB server. Defaults to "https://db.fauna.com:443"</param>
         /// <param name="timeout">Timeout for I/O operations. Defaults to 1 minute.</param>
         /// <param name="httpVersion">Version of http. Default value is HttpVersion.Version11, is you use .net core 3.0 and above you can enable http/2 support by passing HttpVersion.Version20</param>
+        /// <param name="dontCheckNewVersion">Don't check new NuGet package driver version. Default value is null (Check new version).</param>
```

Comment:
You might also want to correct the comments and PR description for "checkNewVersion"

Reply:
Ooh, sure. My bad. I forgot to modify parameter name.